cristin-project/src/test/java/no/unit/nva/cristin/projects/query/organization/QueryCristinOrganizationProjectHandlerTest.java

```diff
@@ -131,6 +143,42 @@ void shouldReturnSearchResponseWithEmptyHitsWhenBackendFetchIsEmpty() throws Api
         assertThat(0, equalTo(searchResponse.getHits().size()));
     }
 
+    @Test
+    void shouldAddParamsToCristinQueryForFilteringAndReturnOk() throws IOException, ApiGatewayException {
+        cristinApiClient = spy(cristinApiClient);
+        doReturn(new HttpResponseFaker(EMPTY_LIST_STRING,
+                HttpURLConnection.HTTP_OK, generateHeaders(ZERO_VALUE, LINK_EXAMPLE_VALUE)))
+                .when(cristinApiClient).listProjects(any());
+        handler = new QueryCristinOrganizationProjectHandler(cristinApiClient, new Environment());
+        var queryParams = Map.of("funding", FUNDING_SAMPLE,
+                "biobank", BIOBANK_SAMPLE,
+                "keyword", KEYWORD_SAMPLE,
+                "results", "5",
+                "unit", DUMMY_UNIT_ID,
+                "sort", START_DATE);
+        handler.handleRequest(generateHandlerProRealisticRequest(queryParams), output, context);
+        var captor = ArgumentCaptor.forClass(URI.class);
+
+        verify(cristinApiClient).listProjects(captor.capture());
+        var actualURI = captor.getValue().toString();
+        assertThat(actualURI,
+                containsString("page=5"));
+        assertThat(actualURI,
+                containsString("&biobank=" + BIOBANK_SAMPLE));
```

Comment:
For later: 
```
private static final String QUERY_PARAM_URI_TEMPLATE = "&%s=%s;

private static String formattedUriParam(String key, String value) {
  return String.format(QUERY_PARAM_URI_TEMPLATE, key, value)
}

...containsString(formattedUriParam(BIOBANK_ID, BIOBANK_SAMPLE)
```

Or maybe this one could be used, comparing the whole map in one assert?

https://github.com/BIBSYSDEV/nva-cristin-service/blob/main/cristin-commons/src/main/java/no/unit/nva/utils/UriUtils.java

Method `formatQueryParameters` , used by `queryParameters`

Reply:
queryParameters method do a sort so the ordering should always be the same